webapp/src/components/App/Tasks/JobTableView.tsx

```diff
@@ -40,12 +40,27 @@ import RefreshIcon from "@mui/icons-material/Refresh";
 import ArrowDropUpIcon from "@mui/icons-material/ArrowDropUp";
 import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown";
 import { grey } from "@mui/material/colors";
-import { TaskView, TaskType } from "../../../common/types";
+import { TaskView } from "../../../common/types";
 import usePromiseWithSnackbarError from "../../../hooks/usePromiseWithSnackbarError";
 import { getLauncherMetrics } from "../../../services/api/study";
 import LinearProgressWithLabel from "../../common/LinearProgressWithLabel";
 import UsePromiseCond from "../../common/utils/UsePromiseCond";
 import { useInterval } from "react-use";
+import { TaskType } from "../../../services/api/tasks/constants";
+
+const FILTER_LIST: Array<TaskView["type"]> = [
+  "DOWNLOAD",
+  "LAUNCH",
+  TaskType.Copy,
+  TaskType.Archive,
+  TaskType.Unarchive,
+  TaskType.UpgradeStudy,
+  TaskType.ThermalClusterSeriesGeneration,
+  TaskType.Scan,
+  "UNKNOWN",
+];
+
+type FilterListType = (typeof FILTER_LIST)[number];
```

Comment:
is this legacy code ? maybe you can use the `Task` enum from my previous comment

Reply:
Yes it's legacy code, but "DOWNLOAD", "LAUNCH" and "UNKNOWN" are not real task types, this is why I didn't include them in the enum, they just use as filter in the tasks view.

Reply:
ok I understand